plot.py

```diff
@@ -15,8 +15,8 @@
 class Map:
     def __init__(self,tweets_file, dist_dir):
         self.tweets_file = tweets_file
-        self.geolocator = Nominatim(user_agent="map ukraine")
-        self.m = folium.Map(location = (48.3794, 31.1656), zoom_start=6)
+        self.geolocator = Nominatim(user_agent="LiveActionMap")
+        self.m = folium.Map(location=[49.422181441686966, 32.04020925216723], zoom_start=5.5, tiles="Stamen Toner")
```

Comment:
Google tells me the coordinates are 48.3794, 31.1656. Any  source for the location? Plus changing the tiles changes the text to English which is something we don't want. I like the idea of a new user_agent and the summary. Can you remove the folium.Map line and I'll merge the other changes.

Reply:
What if I change the coords and tiles but keep the zoom, is that agreeable?

Reply:
Or this tileset, which is aesthetically pleasing and using the native language?

Reply:
https://leaflet-extras.github.io/leaflet-providers/preview/#filter=Stadia.AlidadeSmoothDark

Reply:
nvm its paid

Reply:
Zoom level is an int, zoom=5.5 defaults to 6. It's also okay to keepthe default tile.

Reply:
k